diff --git a/src/modules/Core.js b/src/modules/Core.js
index 237150c0f7204285527dbbb5a3c0aeb32452e03a..3a6c04a509d23c8d044d6eaf67223104f5d47bd9 100644
--- a/src/modules/Core.js
+++ b/src/modules/Core.js
@@ -612,7 +612,7 @@ export default class Core {
     // if user has not defined a custom function for selection - we handle the brush chart
     // otherwise we leave it to the user to define the functionality for selection
     if (typeof w.config.chart.events.selection !== 'function') {
-      let targets = Array.isArray(w.config.chart.brush.targets) || [
+      let targets = Array.isArray(w.config.chart.brush.targets) ? w.config.chart.brush.targets : [
         w.config.chart.brush.target,
       ]
       // retro compatibility with single target option
